client/src/Components/App/App.js

```diff
@@ -54,7 +54,7 @@ const App = () => {
       <Switch>
         <Route exact path='/'><Home /></Route>
         <Route exact path='/how-are-you-feeling'><Feelings setFeeling={setFeeling}/></Route>
-        <Route exact path='/why-are-you-feeling-that-way'><JournalPrompt updateJournal={saveNewEntry}/></Route>
+        <Route exact path='/why-are-you-feeling-that-way'><JournalPrompt feeling={feeling} updateJournal={saveNewEntry}/></Route>
```

Comment:
Love this addition. I really liked being able to see the feeling I selected in the URL path!